### ISSUE TITLE
Missing requirement /lib/core/agent in Nodetime

### ISSUE THREAD

**Issue by petermein** [lelylan/nodes#6]
Hi,

I'm trying to setup the complete lelylan stack. Its been quite the challenge but i'm almost done. 
One of the last thing is im trying into some problems running the mqtt nodes.

The nodes package is running in to some requirement failures in the nodetime package. Below you can find the output.

Node -v v0.10.41
Npm -v 2.14.17

```
09:36:56 web.1 |  module.js:340
09:36:56 web.1 |      throw err;
09:36:56 web.1 |            ^
09:36:56 web.1 |  Error: Cannot find module './lib/core/agent'
09:36:56 web.1 |      at Function.Module._resolveFilename (module.js:338:15)
09:36:56 web.1 |      at Function.Module._load (module.js:280:25)
09:36:56 web.1 |      at Module.require (module.js:364:17)
09:36:56 web.1 |      at require (module.js:380:17)
09:36:56 web.1 |      at Object.<anonymous> (/home/x/nodes/node_modules/nodetime/index.js:1:80)
09:36:56 web.1 |      at Module._compile (module.js:456:26)
09:36:56 web.1 |      at Object.Module._extensions..js (module.js:474:10)
09:36:56 web.1 |      at Module.load (module.js:356:32)
09:36:56 web.1 |      at Function.Module._load (module.js:312:12)
09:36:56 web.1 |      at Module.require (module.js:364:17)
```

Also when looking for the package and running npm i noticed nodetime is discontinued. Maybe this can be the source of this behaviour?
Source: https://www.npmjs.com/package/nodetime

All in all i'm impressed so far and really looking forward to running and contributing to lelylan.

**Comment by petermein** [comment]
After some inspection i think the nodetime should not be in the node package. I added a PR to remove the dependency and the account specific details.

**Comment by FedeG** [comment]
@andreareginato can you see this issue please? (and #7)
I have this problem in docker image.

**Comment by FedeG** [comment]
this issue is fixed in #7

**Comment by andreareginato** [comment]
Feel free to reopen the issue if needed.
